api/src/main/java/org/openmrs/validator/PatientProgramValidator.java

```diff
@@ -73,9 +73,7 @@ public boolean supports(Class<?> c) {
 	 */
 	@Override
 	public void validate(Object obj, Errors errors) {
-		if (log.isDebugEnabled()) {
-			log.debug(this.getClass().getName() + ".validate...");
-		}
+		log.debug(this.getClass().getName(), "{}.validate...");
```

Comment:
What argument are you passing in here?

Reply:
@dkayiwa , i have made some change in the code.